### DISCUSSION TITLE
Github/Local Downloads (Non-functional)

### DISCUSSION THREAD

**Discussion by ArcticDruid** [Bubberstation/Bubberstation#2051] [codebase discussion]
I have confirmed an current issue.
Cloning/Locally downloading the Bubber Master file, results in some files missing.
There is a constant error string that causes the Build.Batch to fail.

The issue seams to originate from the main file.
I have checked all local issues that can occure, and non seam to be the root infliction.
Running on stable BYOND 515.

![image_2024-09-05_103314302](https://github.com/user-attachments/assets/2a4d29cb-a616-4715-bfbf-4adff53b3f16)
It would be appreciated if someone smarter then me can look into this further.

**Comment by Zonespace27** [comment]
Downgrade to 515.1642
